src/main/java/cl/transbank/util/HttpUtilImpl.java

```diff
@@ -188,13 +189,15 @@ private String formatUrl(String url, String query) {
         return String.format("%s%s%s", url, separator, query);
     }
 
-    private static String getResponseBody(InputStream responseStream) throws IOException {
+    private static String getResponseBody(InputStream responseStream) {
         try (final Scanner scanner = new Scanner(responseStream, StandardCharsets.UTF_8.name())) {
             final String responseBody = scanner.useDelimiter("\\A").next();
             responseStream.close();
 
             logger.log(Level.FINE, String.format("HTTP Response Body : %s", responseBody));
             return responseBody;
+        } catch (Exception e) {
+            return null;
```

Comment:
@goncafa Maybe you should propagate this error wrapped in a RuntimeException rather than just swallow it silently. Otherwise you will give misleading information to the SDK users (e.g: "Unspecified message by Webpay API" when maybe the TCP connection got interrupted?)